namespaces/apollo-one-shot/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: apollo-one-shot
```

Comment:
Can you change the name to something like "output-info" or something

Reply:
Changed!